patch-package
--- a/node_modules/quill/dist/quill.js
+++ b/node_modules/quill/dist/quill.js
@@ -5312,7 +5312,7 @@ var Link = function (_Inline) {
 Link.blotName = 'link';
 Link.tagName = 'A';
 Link.SANITIZED_URL = 'about:blank';
-Link.PROTOCOL_WHITELIST = ['http', 'https', 'mailto', 'tel'];
+Link.PROTOCOL_WHITELIST = ['http', 'https', 'mailto', 'tel', 'file'];
 
 function _sanitize(url, protocols) {
   var anchor = document.createElement('a');
@@ -7115,6 +7115,7 @@ var ScrollBlot = /** @class */ (function (_super) {
         var _this = _super.call(this, node) || this;
         _this.scroll = _this;
         _this.observer = new MutationObserver(function (mutations) {
+            if (_this.domNode.contentEditable === 'false') return // do not process mutations if the editor is disabled
             _this.update(mutations);
         });
         _this.observer.observe(_this.domNode, OBSERVER_CONFIG);
@@ -9741,6 +9742,7 @@ var Tooltip = function () {
     key: 'position',
     value: function position(reference) {
       var left = reference.left + reference.width / 2 - this.root.offsetWidth / 2;
+      left = left > 2 ? left : 2
       // root.scrollTop should be 0 if scrollContainer !== root
       var top = reference.bottom + this.quill.root.scrollTop;
       this.root.style.left = left + 'px';
